hobbit-gui/gui-serverbackend/src/main/java/de/usu/research/hobbit/gui/rest/ExperimentsResources.java

```diff
@@ -65,28 +67,81 @@
         if (Application.isUsingDevDb()) {
             return getDevDb().queryExperiments(ids, challengeTaskId);
         } else {
+            // get user
+            UserInfoBean userInfo = InternalResources.getUserInfoBean(sc);
+            List<SystemBean> userSystems = PlatformControllerClientSingleton.getInstance().requestSystemsOfUser(userInfo.getPreferredUsername());
```

Comment:
But in the end it's gonna be used in all 3 cases here, so why not have it there from the beginning?